azure-pipelines.yml

```diff
@@ -70,7 +70,7 @@ parameters:
   - name: 'PRODUCTION_DEPLOY_TYPE'
     displayName: 'Method to achieve deployment in Production (if enabled):'
     type: string
-    default: deployToStagingSlot
```

Comment:
can we hardcode this since it is not supported on linux ?

Reply:
On linux is not supported autoswap option, but we can deploy on staging slot.
I think that we can maintain this option so if we need to test something on staging we can do it.